src/app.ts

```diff
@@ -90,7 +95,9 @@ export default (opt: {
 		mouseDeltaPos: new Vec2(0),
 		keyState: new ButtonState<Key>(),
 		mouseState: new ButtonState<MouseButton>(),
-		gamepadButtonState: new ButtonState<GamepadButton>(),
+		mergedGamepadState: new GamepadState(),
```

Comment:
I think it's more intuitive to have the global gamepad apis default to gamepads[0] instead of a merged state, wdyt?

Reply:
Hmm I think gamepads[0] is more for detect the real gamepads of the user, I imagine the user's side trying to get the first gamepad using `[0]` and the second using `[1]`, and see how it's possible trigger both with the second controller. But if you want, I can change it, obviously.

Reply:
What do you mean by get the real gamepads of the user and trigger both with the second controller?

I was imagining `isGamepadPressed()` to be a convenience function for `getGamepads()[0].isPressed()` instead of checking "if any gamepad has a button pressed", mainly because I can't think of a case where a game actually want to use a merged gamepad state when multiple gamepads are connected. Not sure, maybe let's merge this for now and decide later.

Reply:
Hmm, yes, maybe decide it later, I think there can be some persons who will want to read the input of all gamepads, for example for one player games. Anyway, both sounds good for me, use `isGamepadXXXX` and `onGamepadXXXX` as alias for `gamepads[0].isXXXX`.